README.md

```diff
@@ -22,20 +22,57 @@ Use the `access_token` from Chrome to do requests
 ### Create new user
 POST /api/users/signup (*No access token in header needed for this request*)
 ```
-    {
-        "token": "<ACCESS_TOKEN_FROM_CHROME"
-    }
+{
+	"token": "<ACCESS_TOKEN_FROM_CHROME>"
+}
 ```
 returns 
-  ```
-    {
-      "data": {
-  	  	"id": <TWAIN_USER_ID>,
-    		"first_name": "<user_first_name>",
-    		"last_name": "<user_last_name>",
-    		"picture_url": "<profile_picture_url_from_google",
-    		"created_at": <CREATED_AT_TIMESTAMP>
-      },
-      "error": "<ERROR_MESSAGE>",
+```	
+{
+	"data": {
+      "id": <TWAIN_USER_ID>,
+      "first_name": "<user_first_name>",
+      "last_name": "<user_last_name>",
+      "picture_url": "<profile_picture_url_from_google",
+      "created_at": <CREATED_AT_TIMESTAMP>
+	},
+	"error": "<ERROR_MESSAGE>",
+}
+```
+
+### Create new task
+POST /api/tasks/
+```
+{
+  "name": "<task_name>"
+  "description": "<task_description>",
+  "duration": <task_duration_in_seconds>
+}
+```
+returns
+```
+{
+	"data": {
+		"id": <task_id>
+	}
+    "error": "<ERROR_MESSAGE>"
+}
+```
+
```

Comment:
Can we actually just return the whole task here? So this is a README and API change. (Also fix formatting here)